src/Components/Facility/AssetCreate.tsx

```diff
@@ -270,6 +270,7 @@ const AssetCreate = (props: AssetProps) => {
             assets: { style: "text-gray-200 pointer-events-none" },
             [assetId || "????"]: { name },
           }}
+          backUrl={`/facility/${facilityId}`}
```

Comment:
Suppose I'm updating an asset (eg: https://deploy-preview-3637--care-net.netlify.app/facility/ba2f8df6-20e6-4351-b2a3-065d7886e11a/assets/88194eaf-491e-4b06-a70f-98c321a001c8) and press back, ideally I should go back to it's asset manage page (https://deploy-preview-3637--care-net.netlify.app/assets/88194eaf-491e-4b06-a70f-98c321a001c8) instead of the facility page it belongs to. Maybe use a ternary operator to check if its create or update operation.

Reply:
https://github.com/coronasafe/care_fe/redefined-
github/pull/3637/commits/85cc6d36e26ab8b3fa99bcffbb73c9a94b708df8

Reply:
[`85cc6d3` (#3637)](https://github.com/coronasafe/care_fe/pull/3637/commits/85cc6d36e26ab8b3fa99bcffbb73c9a94b708df8)